packages/quick-edit/package.json

```diff
@@ -11,7 +11,7 @@
 	"author": "workers-devprod@cloudflare.com",
 	"scripts": {
 		"custom:build": "./build.sh",
-		"dev": "concurrently '../../vendor/vscode/scripts/code-web.sh --extensionDevelopmentPath ../../packages/quick-edit/web/quick-edit-extension' 'npm --prefix web/quick-edit-extension run watch-web' 'yarn --cwd ../../vendor/vscode watch' 'yarn --cwd ../../vendor/vscode watch-web'",
+		"dev": "concurrently 'pnpm exec wrangler pages dev ./web' 'npm --prefix web/quick-edit-extension run watch-web' 'yarn --cwd ../../vendor/vscode watch' 'yarn --cwd ../../vendor/vscode watch-web'",
```

Comment:
So we have swapped out a built in web server for a local pages server?
Anything more to it than that?

Reply:
Yep, that's all that changed here